csrc/mmdeploy/preprocess/transform/crop.cpp

```diff
@@ -62,6 +63,13 @@ Result<Value> CenterCropImpl::Process(const Value& input) {
     }
 
     SetTransformData(output, key, std::move(dst_tensor));
+
+    // trace static info & runtime args
+    if (fuse_transform_ == true) {
```

Comment:
This condition in all modified transform op is unnecessary since we can still trace parameters even if `elena` is not enabled.